tests/integration/config.go

```diff
@@ -54,6 +53,37 @@ type TestRun struct {
 	localSdkPath     string
 }
 
+// TODO: check races
+var defaultValidators = map[validatorID]ValidatorConfig{
```

Comment:
Is it possible to make this a function returning a variable? Not a fan of global mutable `var`s

Reply:
Yes. In the making.